ros/src/util/packages/runtime_manager/scripts/computing.yaml

```diff
@@ -530,35 +530,106 @@ subs :
+#    - name : dp_planner (OpenPlanner - old ver)
+#      desc : Local Planner for OpenPlanner
+#      subs :
+#      - name : dp_planner
+#        desc : for behavior and path planning
+#        cmd  : roslaunch dp_planner dp_planner.launch
+#        param: dp_planner
+#      - name : ff_waypoint_follower
+#        desc : Feedforward PID waypoint follower
+#        cmd  : roslaunch ff_waypoint_follower ffwaypointfollower_car.launch
+#        param: ff_waypoint_follower
     - name : OpenPlanner - Local planning
       desc : Local Planner for OpenPlanner
       subs :
+      - name : op_common_params
+        desc : load common local planning parameters
+        cmd  : roslaunch op_local_planner op_common_params.launch
+        param: op_common_params
       - name : op_trajectory_generator
         desc : generating candidate rollout trajectory from global path
         cmd  : roslaunch op_local_planner op_trajectory_generator.launch
         param: op_trajectory_generator
+      - name : op_motion_predictor
+        desc : predict behavior and trajectories for moving vehicles
+        cmd  : roslaunch op_local_planner op_motion_predictor.launch
+        param: op_motion_predictor
+      - name : op_trajectory_evaluator
+        desc : evaluate candidate rollout trajectory against the tracked obstacles
+        cmd  : roslaunch op_local_planner op_trajectory_evaluator.launch
+        param: op_trajectory_evaluator
+      - name : op_behavior_selector
+        desc : Decision making by selecting best behavior for local planner 
+        cmd  : roslaunch op_local_planner op_behavior_selector.launch
+
+    - name : OpenPlanner - Utilities
+      desc : Testing UI, Loggers , ..
+      subs :
+      - name : op_pose2tf
+        desc : Publish transformation from ndt_pose read from rosbag file 
+        cmd  : roslaunch op_utilities op_pose2tf.launch
+        param: op_pose2tf
+      - name : op_bag_player
+        desc : Send step by step simulation signal, read ready for localization rosbags and navigate them frame by frame
+        cmd  : roslaunch op_utilities op_bag_player.launch
+        param: op_bag_player
+      - name : op_data_logger
+        desc : Compare Results and Log useful information
+        cmd  : roslaunch op_utilities op_data_logger.launch
+        param: op_data_logger
+    - name : OpenPlanner - Simulator
+      desc : Car, Signs, Detection Simulation for OpenPlanner Test
+      subs :
+      - name : op_perception_simulator
+        desc : Simulate euclidean cluster for LIDAR based object detection of simulated cars
+        cmd  : roslaunch op_simulation_package op_perception_simulator.launch
+        param: op_perception_simulator
+      - name : op_signs_simulator
+        desc : Simulate traffic Signs status for intersections included in the map
+        cmd  : roslaunch op_simulation_package op_signs_simulator.launch
+        param: op_signs_simulator
+      - name : op_car_simulator_1
```

Comment:
@hatem-darweesh There are 5 checkboxes for op_car_simulator. This should be only one here.

Reply:
@hatem-darweesh How about @kitsukawa's comment?

Reply:
@kitsukawa @kfunaoka Only 1 check box for simulated car remaining, I will add multi car simulation option in "Simulation" tab later.